src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -437,31 +423,26 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
         <Announced
           message={`${sampleQueries.length} search results available.`}
         />
-        <div role='navigation'
-          onMouseEnter={() => this.setState({ isHoverOverSampleQueriesList: true })}
-          onMouseLeave={() => this.setState({ isHoverOverSampleQueriesList: false })}>
-          <DetailsList
-            styles={this.state.isHoverOverSampleQueriesList ? { root: { overflow: 'scroll' } } : { root: { overflow: 'hidden' } }}
-            className={classes.queryList}
-            cellStyleProps={{
-              cellRightPadding: 0,
-              cellExtraRightPadding: 0,
-              cellLeftPadding: 0,
-            }}
-            onRenderItemColumn={this.renderItemColumn}
-            items={sampleQueries}
-            selectionMode={SelectionMode.none}
-            columns={columns}
-            groups={groups}
-            groupProps={{
-              showEmptyGroups: true,
-              onRenderHeader: this.renderGroupHeader,
-            }}
-            onRenderRow={this.renderRow}
-            onRenderDetailsHeader={this.renderDetailsHeader}
-            onItemInvoked={this.querySelected}
-          />
-        </div>
+        <DetailsList
```

Comment:
We need the div with the navigation role for accessibility purposes

Reply:
aaah I see